sechub-sereco/src/test/java/com/daimler/sechub/sereco/test/AssertVulnerabilities.java

```diff
@@ -175,6 +190,46 @@ public VulnerabilityFinder withType(String type) {
             search.setType(type);
             return this;
         }
+        
+        public FindCodeCallStackBuilder withCodeLocation(String location, int line, int column) {
+            return new FindCodeCallStackBuilder(location,line,column);
+        }
+        
+        public class FindCodeCallStackBuilder{
+            
+            private SerecoCodeCallStackElement currentCallStackElement;
+
+            public FindCodeCallStackBuilder(String location, int line, int column) {
+                currentCallStackElement = createCallStackElement(location, line, column);
+                
+                search.setCode(currentCallStackElement);
+            }
+            
+            public FindCodeCallStackBuilder calling(String location, int line, int column) {
+                SerecoCodeCallStackElement childCallstackElement =createCallStackElement(location, line, column);
```

Comment:
Please add a space after the equals sign.

```suggestion
                SerecoCodeCallStackElement childCallstackElement = createCallStackElement(location, line, column);
```

Reply:
done: formatted source code